slides/design-burst-week7.html

```diff
@@ -0,0 +1,185 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no"
+    />
+
+    <title>Design burst - Week 7</title>
+
+    <link rel="stylesheet" href="../css/reveal.css" />
+    <link rel="stylesheet" href="../css/theme/fac.css" />
+
+    <!-- Theme used for syntax highlighting of code -->
+    <link rel="stylesheet" href="../lib/css/zenburn.css" />
+
+    <!-- Printing and PDF exports -->
+    <script>
+      var link = document.createElement("link");
+      link.rel = "stylesheet";
+      link.type = "text/css";
+      link.href = window.location.search.match(/print-pdf/gi)
+        ? "../css/print/pdf.css"
+        : "../css/print/paper.css";
+      document.getElementsByTagName("head")[0].appendChild(link);
+    </script>
+
+    <style>
+      /* BLACK: #121212 */
+      /* YELLOW: #ffe500 */
+      /* BLUE: #0805ec */
+      /* GREEN: #00be00 */
+      /* RED: #ff2800 */
+    </style>
+  </head>
+
+  <body>
+    <div class="reveal">
+      <div class="slides">
+        <section>
+          <img src="../assets/logos/logo-mobile.png" class="img-s" />
+          <h2>Design burst</h2>
+          <h3>Week 7: Error message design</h3>
+          <aside class="notes">
+            Sources: https://cxl.com/blog/error-messages/
+          </aside>
+        </section>
+        <section>
+          <section>
+            <h3><span class="bigunderline">1. Set clear expectations</span></h3>
+            <img src="https://i.imgur.com/50HVp96.png" />
+            <aside class="notes">
+              Source: https://material.io/components/text-fields/#anatomy
+            </aside>
+          </section>
+          <section>
+            <p>
+              Use helper text to convey additional guidance about form input
+              fields
+            </p>
+            <img src="https://i.imgur.com/Gsi1vNw.png" />
+          </section>
+          <section>
+            <p>
+              When text input isn’t accepted, an error message can display
+              instructions on how to fix it
+            </p>
+            <img src="https://i.imgur.com/axfy8KS.png" />
```

Comment:
I'm not convinced these Google screenshots are that helpful. I would rather show examples we think are good. E.g. this one doesn't show instructions on how to fix the problem